src/Orleans.Core/Runtime/OutsideRuntimeClient.cs

```diff
@@ -132,10 +132,10 @@ internal void ConsumeServices(IServiceProvider services)
 
             this.GrainReferenceRuntime = this.ServiceProvider.GetRequiredService<IGrainReferenceRuntime>();
 
-            var statisticsOptions = this.ServiceProvider.GetRequiredService<IOptions<StatisticsOptions>>();
+            var statisticsOptions = this.ServiceProvider.GetRequiredService<IOptions<ClientStatisticsOptions>>();
```

Comment:
I think the general rule of thumb is to store `IOption<TOptions>`.Value, instead of `IOption<TOPtions>`

Reply:
We don't do this remotely consistently.  Why does this matter?